src/components/m-form-group/index.js

```diff
@@ -0,0 +1,45 @@
+import classnames from 'classnames';
+
+import BaseComponentGlobal from '../../js/abstract/base-component-global';
+import wcdomready from '../../js/wcdomready';
+
+import FormGroup from './js/form-group';
+
+import styles from './index.scss';
+import template from './_template';
+
+class AXAFormGroup extends BaseComponentGlobal {
+  static get observedAttributes() {
+    return ['label', 'info'];
+  }
+
+  constructor() {
+    super(styles, template);
+  }
+
+  connectedCallback() {
```

Comment:
@TheDadi 
For incremental rendering `connectedCallback` should be avoid until  really needed and replaced by `willRenderCallback` and `didRenderCallback`.
I refactored all components recently and documented stuff at https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTING.md#custom-elements